### ISSUE TITLE
Wierd camera position with `FoVPerspectiveCameras`.

### ISSUE THREAD

**Issue by zeakey** [facebookresearch/pytorch3d#863]
I'm trying to rendering a hand mesh follow the official tutorial <https://pytorch3d.org/tutorials/render_textured_meshes>.

However, given exactly the same camera parameters, the camera position seems to be different
when rendering the cow, and rendering my hand model.

Below is the simplified codes:
```
def render_mesh(mesh):
    R, T = look_at_view_transform(2.7, 0, 180)
    cameras = FoVPerspectiveCameras(device=device, R=R, T=T)

    raster_settings = RasterizationSettings(
        image_size=512, 
        blur_radius=0.0, 
        faces_per_pixel=1, 
    )

    lights = PointLights(device=device, location=[[0.0, 0.0, -3.0]])

    renderer = MeshRenderer(
        rasterizer=MeshRasterizer(
            cameras=cameras, 
            raster_settings=raster_settings
        ),
        shader=SoftPhongShader(
            device=device, 
            cameras=cameras,
            lights=lights
        )
    )
    return renderer(mesh)

cow_mesh = load_objs_as_meshes(['data/cow_mesh/cow.obj'], device=device)
verts, faces = cow_mesh.get_mesh_verts_faces(0)
fig = plt.figure(figsize=(10, 10))
ax = fig.add_subplot(221, projection='3d')
draw_mesh(ax, verts.cpu().squeeze(dim=0).numpy(), faces.cpu().squeeze(dim=0).numpy())

images = render_mesh(cow_mesh)
ax = fig.add_subplot(222)
ax.imshow(images[0, ..., :3].cpu().numpy())
ax.axis("off");

hand = torch.load('handmesh.pth')
verts3d = hand['verts3d'].to(device)
verts3d = normalize_verts(verts3d)
faces = hand['faces'].to(device)
hasnmesh = pytorch3d.structures.Meshes(verts3d, faces)

color = torch.ones(1, verts3d.size(1), 3, device=device)
hasnmesh.textures = TexturesVertex(verts_features=color)

ax = fig.add_subplot(223, projection='3d')
draw_mesh(ax, verts3d.squeeze().cpu().numpy(), faces.squeeze().cpu().numpy())

images = render_mesh(hasnmesh)
ax = fig.add_subplot(224)
ax.imshow(images[0, ..., :3].cpu().numpy())
ax.axis("off");
```

The mesh and rendering results as as below:
![企业微信截图_16329021615906](https://user-images.githubusercontent.com/6763078/135226739-c86a21b6-4970-410c-99f6-d84a583d799f.png)

In the case of the cow,  The camera is on the negative half axis of the z-axis, looking at the object.
However, in the case of hand, the camera seems to be on the negative half axis of the y-axis.

**Comment by zeakey** [comment]
After plotting the original `verts`, I found that the problem is that the conversion from `verts` and `faces` to `mesh` switches axis.
![企业微信截图_16329069677717](https://user-images.githubusercontent.com/6763078/135240074-b4fc3441-73c6-4021-8fb9-403a81d01d80.png)
